musicname/ze_mountain_escape_bbo_v1.cfg

```diff
@@ -0,0 +1,16 @@
+"music"
+{
+    "end.mp3"                           "Pendulum - Watercolour"
+    "start.mp3"                         "The Prodigy - Omen"
+    "start2.mp3"                        "Madeon - Finale (Netsky Remix)"
+    "emart.mp3"                         "원곡 - 이마트송"
+    "ffvii_e2_bloodsugar.mp3"           "Pendulum - Blood Sugar"
+    "ffvii_e2_selfvsself.mp3"           "Pendulum - Self vs Self (ft. In Flames)"
+    "ffvii_e2_slam.mp3"                 "Pendulum - Slam (Video)"
```

Comment:
Is `(Video)` necessary for Slam?

Reply:
https://github.com/Whennya/Music-Names/blob/ad5f0f7036f3e30698a7d2738c1ac68210857367/musicname/ze_ffvii_mako_reactor_v5_3.cfg#L24
copy pasted for consistency considering they were the same file, should maybe be removed from ze_ffvii_mako_russianversion134.cfg aswell then?

Reply:
I'm pretty sure keeping (Video) in this instance is correct, since it leads to which version of this song is actually used in the map:
https://youtu.be/gIOQfdn9L9c?si=xpGWDJNDEtf7QT42&t=21 where this is the actual song used
https://www.youtube.com/watch?v=p_HnxvdeK0U this is the original song

Reply:
I'll let you be the judge on what to put.